boards/arm/nrf52840_mdk_m2_module/nrf52840_mdk_m2_module.dts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2022 Matt Anderson
+ * Copyright (c) 2022 Nikola Trifunovic
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "nrf52840_mdk_m2_module-pinctrl.dtsi"
+
+/ {
+	model = "nRF52840 MDK M2 Module";
+	compatible = "nrf52840_mdk_m2_module";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0_red: led_0 {
+			gpios = <&gpio0 30 GPIO_ACTIVE_LOW>;
+			label = "Red LED 1";
+		};
+		led0_green: led_1 {
+			gpios = <&gpio0 29 GPIO_ACTIVE_LOW>;
+			label = "Green LED 2";
+		};
+		led0_blue: led_2 {
+			gpios = <&gpio0 31 GPIO_ACTIVE_LOW>;
+			label = "Blue LED 3";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		red_pwm_led: pwm_led_0 {
+			pwms = <&pwm0 30>;
```

Comment:
I had to add `PWM_MSEC(20) PWM_POLARITY_INVERTED` for all PWM definitions for them to work on a newer Zephyr version.